hail_search/hail_search_query.py

```diff
@@ -318,6 +331,39 @@ def _valid_genotype_family_entries(cls, entries, gentoype_entry_indices, genotyp
         )
         return hl.or_missing(is_valid, entries)
 
+    @classmethod
+    def _get_genotype_passes_quality_filter(cls, quality_filter):
+        affected_only = quality_filter.get('affected_only')
+        passes_quality = None
+        for filter_k, value in quality_filter.items():
+            field = cls.GENOTYPE_FIELDS.get(filter_k.replace('min_', ''))
+            if field and value:
+                passes_quality = cls._get_genotype_passes_quality_field(field, value, affected_only, passes_quality)
+
+        return passes_quality
+
+    @classmethod
+    def _get_genotype_passes_quality_field(cls, field, value, affected_only, filter_func):
+        field_config = cls.QUALITY_FILTER_FORMAT.get(field) or {}
+        if field_config.get('scale'):
+            value = value / field_config['scale']
+
+        def passes_quality_field(gt):
+            is_valid = (gt[field] >= value) | hl.is_missing(gt[field])
+            if field_config.get('override'):
```

Comment:
Use `'override' in field_config` instead of the `get` function?

Reply:
thats not really any more readable, and would add a weird edge case if `override` has null. Not sure what the advantage is?

Reply:
I thought that the `override` was a configuration field. Configuring it to `null` or `false` doesn't make sense because it needs to be a function. The `'key' in dict` checks the key while the `get` function uses both key and value. It saves a bit of performance.
Now, using the `namedtuple` class is all good.